frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -210,6 +210,10 @@ action.GitMachete.FastForwardMergeSelectedToParentAction.description=Fast-forwar
 action.GitMachete.SyncSelectedToParentByMergeAction.text=Git Machete: Checkout and Sync to Parent by Merge
 action.GitMachete.SyncSelectedToParentByMergeAction.GitMacheteContextMenu.text=Checkout and Sync to Parent by _Merge
 action.GitMachete.SyncSelectedToParentByMergeAction.description=Checkout this branch and merge parent into it
+action.GitMachete.SyncSelectedToParentByMergeAction.warning-dialog.title=Merge operation warning
+action.GitMachete.SyncSelectedToParentByMergeAction.warning-dialog.message=Using merge to sync branches is discouraged, especially with stacked PRs. Ignore this message and proceed if this is a hotfix branch
+action.GitMachete.SyncSelectedToParentByMergeAction.warning-dialog.do-not-show-again=Don't show it again
+action.GitMachete.SyncSelectedToParentByMergeAction.warning-dialog.ok-text=Proceed anyway
```

Comment:
Before I test it again... is there a Cancel button in this dialog? 🤔 in case someone decides **not** to proceed

Reply:
Yes, there is, it's an OkCancel dialog. I've just left the default cancel text as it suits the purpose here.
Take a look at `show()` here:
<img width="1160" alt="image" src="https://user-images.githubusercontent.com/61919032/189371962-75bb4852-a8c1-402a-aae5-4de1842c5d8a.png">